build/build.sh

```diff
@@ -52,6 +52,8 @@ then
     --to=docx \
     --filter pandoc-fignos \
     --filter pandoc-tablenos \
+    --filter pandoc-img-glob \
```

Comment:
`--filter pandoc-img-glob`

@dhimmel is restructuring the relative path to the image for docx conversion to absolute paths relative to markdown. Without it I get these errors:

` 
    [pandoc warning] Could not find image "images/image2.png", skipping...
`